examples/medical_text_understanding/medical_text_understanding.py

```diff
@@ -0,0 +1,74 @@
+import sys
+from termcolor import colored
+
+from forte.data.data_pack import DataPack
+from forte.data.readers import PlainTextReader
+from forte.pipeline import Pipeline
+from forte.processors.writers import PackIdJsonPackWriter
+
+from ft.onto.base_ontology import (
+    Token,
+)
+
+from ftx.medical.clinical_ontology import Hyponym, Abbreviation
+from fortex.health.processors.scispacy_processor import (
+    ScispaCyProcessor,
+)
+
+def main(
+    input_path: str,
+    process: str,
+):
+    pl = Pipeline[DataPack]()
+    pl.set_reader(PlainTextReader())
+    if process == "abb":
```

Comment:
Suggetsion: I am thinking lets name these options as "abbreviation" and "hyponym" instead of "abb" and "hyp". It just seems more readable.